src/cli/download_tracker.rs

```diff
@@ -186,13 +186,15 @@ impl DownloadTracker {
                             (remaining / speed) as u64
                         });
                         format!(
-                            "{} / {} ({:3.0} %) {} in {} ETA: {}",
+                            "{} / {} ({:3.0} %) {} in {}{}",
                             total_h,
                             content_len_h,
                             percent,
                             speed_h,
                             elapsed_h.display(),
-                            eta_h.display(),
+                            (eta_h != Duration::ZERO)
+                                .then(|| format!(" ETA: {}", eta_h.display()))
+                                .unwrap_or_default()
```

Comment:
I think I'd prefer to formulate it like this:

```rust
match eta_h {
     Duration::ZERO => format_args!(""),
     _ => format_args!(" ETA: {}", eta_h.display()),
}
```

Reply:
hmm, that doesn't work because the `format_args!` creates a temporary which doesn't survive the match scope